### PR TITLE
[plugin.video.nhlgcl@matrix] 2021.1.21+matrix.1

### PR DESCRIPTION
**Pull request by eracknaphobia**
### Add-on details:

- **General**
  - Add-on name: NHL TV™
  - Add-on ID: plugin.video.nhlgcl
  - Version number: 2021.1.21+matrix.1
  - Kodi/repository version: matrix

- **Code location**
  - URL: https://github.com/eracknaphobia/plugin.video.nhlgcl
  
With NHL.TV you can access revolutionary 60fps (frames per second) video through Kodi
        

### Description of changes:

            - Update code for python 3 compatibility
            - Fix for too many sign-ins error
            - Code Cleanup
        

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.nhlgcl/resources/lib/globals.py [hunk 1]
```diff
@@ -1,28 +1,31 @@
...
 
 addon_handle = int(sys.argv[1])
 
 # Addon Info
 ADDON = xbmcaddon.Addon()
 ADDON_ID = ADDON.getAddonInfo('id')
 ADDON_VERSION = ADDON.getAddonInfo('version')
-ADDON_PATH_PROFILE = xbmcvfs.translatePath(ADDON.getAddonInfo('profile'))
-XBMC_VERSION = float(re.findall(r'\d{2}\.\d{1}', xbmc.getInfoLabel("System.BuildVersion"))[0])
+ADDON_PATH_PROFILE = xbmc.translatePath(ADDON.getAddonInfo('profile'))
```
Comment (enen92) [inline plugin.video.nhlgcl/resources/lib/globals.py:27]: sorry, you have to use xbmcvfs.translatepath for matrix++
xbmc.translatepath will be removed in v20

**Review by enen92 — CHANGES_REQUESTED** [review]
only the translatepath change is required and it's ready to go in

File: plugin.video.nhlgcl/resources/lib/globals.py (hunk 1 above)
Comment (eracknaphobia) [reply plugin.video.nhlgcl/resources/lib/globals.py:27]: I think line 16 through 19 should do the conversion for xbmcvfs

Comment (enen92) [reply plugin.video.nhlgcl/resources/lib/globals.py:27]: You're right

**Review by enen92 — APPROVED** [review]

**Pull request merged**